src/app/Http/Controllers/CreateController.php

```diff
@@ -5,19 +5,51 @@
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\DB;
 use App\Http\Requests\TaskRequest;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Http;
+use App\Models\WeatherTask;
 
 class CreateController extends Controller
 {
     public function __invoke(TaskRequest $request)
     {
         $validated = $request->validated();
+        $weather = null;
+
+        if (
+            Carbon::now()->addDays(5)->gte(Carbon::parse($validated['start'])) &&
+            Carbon::now()->lte(Carbon::parse($validated['start']))
```

Comment:
I think it is backwards, (Today + 5 > X && Today < X),

Which day is higher than today+5 and lower than today?

I think it should be (Today + 5 <= X && Today >= X), for example Yesterday not works and that is fine, and also tomorrow works because is Today+1 and also is higher or equals than today.